### ISSUE TITLE
Adding hyperlink to table cell doesn't work

### ISSUE THREAD

**Issue by tbowmo** [gitbrent/PptxGenJS#1049]
Is it all possible to make hyperlinks in table cells?

TableCellProps doesn't contain a hyperlink property, but is that just because of typedefs that is out of sync, or is it not supported at all?

**Comment by gitbrent** [comment]
Thanks for reporting this.

The issue is with the typescript def for `TableCellProps`

Hyperlinks work fine in a plain JS sandbox:
```javascript
let pptx = new PptxGenJS();
let slide = pptx.addSlide();

slide.addTable(
  [[{ text: "Link with Tooltip", options: { hyperlink: { url: "https://github.com/gitbrent/pptxgenjs", tooltip: "Visit Homepage" } } }]],
  {
	x: 0.5,
	y: 1.1,
	w: 5.0,
	rowH: 0.75,
	fill: { color: "F7F7F7" },
	fontSize: 14,
	color: "363636",
	border: { pt: "1", color: "BBCCDD" },
  }
);

pptx.writeFile({ fileName: "PptxGenJS-Sandbox.pptx" });
```
![Screen Shot 2022-02-24 at 19 13 32](https://user-images.githubusercontent.com/7218970/155634943-aa84864d-1218-4000-a2e4-43cf1d8a4110.png)

**Comment by gitbrent** [comment]
Fixed in v3.10.0
